deploy/remotedeploy.sh

```diff
@@ -31,6 +31,17 @@ if ! sudo cp "$PYSHELL_PATH/pyshell.service" /lib/systemd/system/; then
     exitFailed "Service file copy to systemd failed"
 fi 
 
+if [ -f /etc/os-release ]; then
+    ID=$(grep '^ID=' /etc/os-release | cut -d= -f2 | tr -d '"')
+    ID_LIKE=$(grep '^ID_LIKE=' /etc/os-release | cut -d= -f2 | tr -d '"')
+    if [[ "$ID" == "debian" || "$ID_LIKE" == *"debian"* ]]; then
+        echo "Running on a Debian-based system"
```

Comment:
Change this to be meaningful. You aren't informing that we are on Debian - you are informing that you are installing venv package since we are on a Debian based or Ubuntu system and since it was not detected as already present.